src/main/kotlin/Cell.kt

```diff
@@ -0,0 +1,7 @@
+interface Cell {
+    fun row(): Int
+
+    fun column(): Int
```

Comment:
```suggestion
    val row: Int
    val column: Int
```
게터형태라면, Kotlin의 val 프로퍼티가 조금더 자연스럽진 않을까요?